common/oauthTokenManager.go

```diff
@@ -627,7 +394,7 @@ const TokenRefreshSourceTokenStore = "tokenstore"
 
 // OAuthTokenInfo contains info necessary for refresh OAuth credentials.
 type OAuthTokenInfo struct {
-	azcore.TokenCredential
+	azcore.TokenCredential `json:"-"`
```

Comment:
So the current flow for login is `azcopy login` -> `azcopy whatever` (outside of one-shot login). If we're not persisting the refresh info between runs, then interactive login is completely broken. Are we saving that info some other way?

Reply:
I took a look at the code again and we should be persisting the token between runs. We can discuss the code changes further if you'd like.